File: ProTip365/Onboarding/OnboardingLocalization.swift
import Foundation

// MARK: - Onboarding Localization

struct OnboardingLocalization {
    private let language: String

    init(language: String) {
        self.language = language
    }

    // MARK: - Welcome and General

    var welcomeTitle: String {
        switch language {
        case "fr": return "Configuration initiale"
        case "es": return "Configuración inicial"
        default: return "Initial Setup"
        }
    }

    var backButtonText: String {
        switch language {
        case "fr": return "Retour"
        case "es": return "Atrás"
        default: return "Back"
        }
    }

    var nextButtonText: String {
        switch language {
        case "fr": return "Suivant"
        case "es": return "Siguiente"
        default: return "Next"
        }
    }

    var finishButtonText: String {
        switch language {
        case "fr": return "Terminer"
        case "es": return "Finalizar"
        default: return "Finish"
        }
    }

    var skipButtonText: String {
        switch language {
        case "fr": return "Passer"
        case "es": return "Omitir"
        default: return "Skip"
        }
    }

    var cancelButtonText: String {
        switch language {
        case "fr": return "Annuler"
        case "es": return "Cancelar"
        default: return "Cancel"
        }
    }

    func stepTitle(currentStep: Int, totalSteps: Int) -> String {
        switch language {
        case "fr": return "Étape \(currentStep) sur \(totalSteps)"
        case "es": return "Paso \(currentStep) de \(totalSteps)"
        default: return "Step \(currentStep) of \(totalSteps)"
        }
    }

    // MARK: - Step 1: Language

    var languageSelectionTitle: String {
        switch language {
        case "fr": return "Choisissez votre langue"
        case "es": return "Elige tu idioma"
        default: return "Choose Your Language"
        }
    }

    var languageStepDescription: String {
        switch language {
        case "fr": return "Dans quelle langue préférez-vous utiliser l'application?"
        case "es": return "¿En qué idioma prefieres usar la aplicación?"
        default: return "What language would you prefer to use the app in?"
        }
    }

    // MARK: - Step 2: Multiple Employers

    var multipleEmployersTitle: String {
        switch language {
        case "fr": return "Employeurs multiples"
        case "es": return "Múltiples empleadores"
        default: return "Multiple Employers"
        }
    }

    var multipleEmployersQuestion: String {
        switch language {
        case "fr": return "Travaillez-vous pour plusieurs employeurs?"
        case "es": return "¿Trabajas para múltiples empleadores?"
        default: return "Do you work for multiple employers?"
        }
    }

    var multipleEmployersStepDescription: String {
        switch language {
        case "fr": return "Aidez-nous à personnaliser l'application pour votre situation de travail"
        case "es": return "Ayúdanos a personalizar la aplicación para tu situación laboral"
        default: return "Help us customize the app for your work situation"
        }
    }

    var multipleEmployersExplanationTitle: String {
        switch language {
        case "fr": return "Pourquoi cette question?"
        case "es": return "¿Por qué esta pregunta?"
        default: return "Why this question?"
        }
    }

    var multipleEmployersExplanation: String {
        switch language {
        case "fr": return "Si vous avez plusieurs employeurs, vous pourrez créer des quarts de travail et définir des entrées par employeur. Cela vous permettra de mieux organiser et analyser vos revenus par source."
        case "es": return "Si tienes múltiples empleadores, podrás crear turnos y definir entradas por empleador. Esto te permitirá organizar y analizar mejor tus ingresos por fuente."
        default: return "If you have multiple employers, you'll be able to create shifts and define entries per employer. This will help you better organize and analyze your income by source."
        }
    }

    var employerNameLabel: String {
        switch language {
        case "fr": return "Nom de votre employeur"
        case "es": return "Nombre de su empleador"
        default: return "Your employer's name"
        }
    }

    var employerNamePlaceholder: String {
        switch language {
        case "fr": return "Ex: Restaurant ABC"
        case "es": return "Ej: Restaurante ABC"
        default: return "e.g. ABC Restaurant"
        }
    }

    var setupEmployersTitle: String {
        switch language {
        case "fr": return "Configurer les employeurs"
        case "es": return "Configurar empleadores"
        default: return "Set Up Employers"
        }
    }

    var doneButtonText: String {
        switch language {
        case "fr": return "Terminé"
        case "es": return "Hecho"
        default: return "Done"
        }
    }

    // MARK: - Step 3: Week Start

    var weekStartTitle: String {
        switch language {
        case "fr": return "Début de semaine"
        case "es": return "Inicio de semana"
        default: return "Week Start Day"
        }
    }

    var weekStartStepDescription: String {
        switch language {
        case "fr": return "Quel jour commence votre semaine de travail? Cela affectera vos rapports hebdomadaires."
        case "es": return "¿Qué día comienza tu semana laboral? Esto afectará tus reportes semanales."
        default: return "What day does your work week start? This will affect your weekly reports."
        }
    }

    var weekStartExplanationTitle: String {
        switch language {
        case "fr": return "Pourquoi cette question?"
        case "es": return "¿Por qué esta pregunta?"
        default: return "Why this matters"
        }
    }

    var weekStartExplanation: String {
        switch language {
        case "fr": return "Ceci définira comment vos rapports hebdomadaires sont calculés et organisés."
        case "es": return "Esto definirá cómo se calculan y organizan tus reportes semanales."
        default: return "This will define how your weekly reports are calculated and organized."
        }
    }

    func localizedWeekDay(_ index: Int) -> String {
        let weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
        switch language {
        case "fr":
            return ["Dimanche", "Lundi", "Mardi", "Mercredi", "Jeudi", "Vendredi", "Samedi"][index]
        case "es":
            return ["Domingo", "Lunes", "Martes", "Miércoles", "Jueves", "Viernes", "Sábado"][index]
        default:
            return weekDays[index]
        }
    }

    // MARK: - Step 4: Security

    var securityTitle: String {
        switch language {
        case "fr": return "Sécurité"
        case "es": return "Seguridad"
        default: return "Security"
        }
    }

    var securityStepDescription: String {
        switch language {
        case "fr": return "Sécurisez l'accès à vos données financières sensibles"
        case "es": return "Asegura el acceso a tus datos financieros sensibles"
        default: return "Secure access to your sensitive financial data"
        }
    }

    var securityExplanationTitle: String {
        switch language {
        case "fr": return "Sécurité supplémentaire"
        case "es": return "Seguridad adicional"
        default: return "Additional Security"
        }
    }

    var securityExplanation: String {
        switch language {
        case "fr": return "En plus de votre nom d'utilisateur et mot de passe, vous pouvez ajouter une couche de sécurité supplémentaire pour restreindre l'accès à l'application."
        case "es": return "Además de tu nombre de usuario y contraseña, puedes agregar una capa adicional de seguridad para restringir el acceso a la aplicación."
        default: return "On top of your username and password, you can add an additional layer of security to restrict access to the app."
        }
    }

    var faceIDText: String {
        switch language {
        case "fr": return "Face ID / Touch ID"
        case "es": return "Face ID / Touch ID"
        default: return "Face ID / Touch ID"
        }
    }

    var pinCodeText: String {
        switch language {
        case "fr": return "Code PIN"
        case "es": return "Código PIN"
        default: return "PIN Code"
        }
    }

    var noSecurityText: String {
        switch language {
        case "fr": return "Aucune"
        case "es": return "Ninguna"
        default: return "None"
        }
    }

    // MARK: - Step 5: Variable Schedule

    var variableScheduleTitle: String {
        switch language {
        case "fr": return "Horaire variable"
        case "es": return "Horario variable"
        default: return "Variable Schedule"
        }
    }

    var variableScheduleQuestion: String {
        switch language {
        case "fr": return "Avez-vous un horaire de travail variable?"
        case "es": return "¿Tienes un horario de trabajo variable?"
        default: return "Do you have a variable work schedule?"
        }
    }

    var variableScheduleStepDescription: String {
        switch language {
        case "fr": return "Aidez-nous à configurer vos objectifs en fonction de votre horaire"
        case "es": return "Ayúdanos a configurar tus objetivos según tu horario"
        default: return "Help us set up your goals based on your schedule"
        }
    }

    var variableScheduleExplanationTitle: String {
        switch language {
        case "fr": return "Pourquoi cette question?"
        case "es": return "¿Por qué esta pregunta?"
        default: return "Why this question?"
        }
    }

    var variableScheduleExplanation: String {
        switch language {
        case "fr": return "Si vous avez un horaire variable, nous ne configurerons que des objectifs quotidiens au lieu d'objectifs hebdomadaires et mensuels, car ceux-ci sont plus pertinents pour les horaires fixes et définis."
        case "es": return "Si tienes un horario variable, solo configuraremos objetivos diarios en lugar de objetivos semanales y mensuales, ya que estos son más relevantes para horarios fijos y definidos."
        default: return "If you have a variable schedule, we will only set daily targets instead of weekly and monthly targets, as those are more relevant for fixed, defined schedules."
        }
    }

    // MARK: - Step 6: Sales and Tip Targets

    var salesAndTipTargetsTitle: String {
        switch language {
        case "fr": return "Objectifs et cibles"
        case "es": return "Objetivos y metas"
        default: return "Goals & Targets"
        }
    }

    var salesAndTipTargetsStepDescription: String {
        switch language {
        case "fr": return "Définissez vos objectifs quotidiens pour les pourboires, les ventes et les heures"
        case "es": return "Establece tus objetivos diarios para propinas, ventas y horas"
        default: return "Set your daily goals for tips, sales, and hours"
        }
    }

    var tipPercentageTargetLabel: String {
        switch language {
        case "fr": return "Pourcentage de pourboire cible"
        case "es": return "Porcentaje objetivo de propinas"
        default: return "Target Tip Percentage"
        }
    }

    var tipPercentageNoteTitle: String {
        switch language {
        case "fr": return "Pourcentage des ventes"
        case "es": return "Porcentaje de ventas"
        default: return "Percentage of Sales"
        }
    }

    var tipPercentageNoteMessage: String {
        switch language {
        case "fr": return "Définissez votre objectif de pourboire en pourcentage de vos ventes totales. Par exemple, 15% signifie que vous visez 15% de vos ventes en pourboires."
        case "es": return "Establezca su objetivo de propinas como porcentaje de sus ventas totales. Por ejemplo, 15% significa que apunta a 15% de sus ventas en propinas."
        default: return "Set your tip goal as a percentage of your total sales. For example, 15% means you aim for 15% of your sales in tips."
        }
    }

    var averageDeductionPercentageLabel: String {
        switch language {
        case "fr": return "Taux d'impôt moyen"
        case "es": return "Tasa de impuestos promedio"
        default: return "Average Income Tax Rate"
        }
    }

    var averageDeductionPercentageNoteTitle: String {
        switch language {
        case "fr": return "Impôt sur le revenu"
        case "es": return "Impuesto sobre la renta"
        default: return "Income Tax"
        }
    }

    var averageDeductionPercentageNoteMessage: String {
        switch language {
        case "fr": return "Taux d'impôt moyen sur votre revenu total. Utilisé pour calculer votre revenu net estimé après impôts."
        case "es": return "Tasa promedio de impuestos sobre sus ingresos totales. Se usa para calcular sus ingresos netos estimados después de impuestos."
        default: return "Average income tax rate on your total income. Used to calculate your estimated net income after taxes."
        }
    }

    var dailySalesTargetLabel: String {
        switch language {
        case "fr": return "Ventes prévue/jour"
        case "es": return "Ventas diarias"
        default: return "Daily Sales"
        }
    }

    var weeklySalesTargetLabel: String {
        switch language {
        case "fr": return "Ventes hebdomadaires"
        case "es": return "Ventas semanales"
        default: return "Weekly Sales"
        }
    }

    var monthlySalesTargetLabel: String {
        switch language {
        case "fr": return "Ventes mensuelles"
        case "es": return "Ventas mensuales"
        default: return "Monthly Sales"
        }
    }

    var dailyHoursTargetLabel: String {
        switch language {
        case "fr": return "# d'heures/jour"
        case "es": return "Horas diarias"
        default: return "Daily Hours"
        }
    }

    var weeklyHoursTargetLabel: String {
        switch language {
        case "fr": return "Heures hebdomadaires"
        case "es": return "Horas semanales"
        default: return "Weekly Hours"
        }
    }

    var monthlyHoursTargetLabel: String {
        switch language {
        case "fr": return "Heures mensuelles"
        case "es": return "Horas mensuales"
        default: return "Monthly Hours"
        }
    }

    var variableScheduleNoteTitle: String {
        switch language {
        case "fr": return "Horaire variable"
        case "es": return "Horario variable"
        default: return "Variable Schedule"
        }
    }

    var variableScheduleNoteMessage: String {
        switch language {
        case "fr": return "Puisque vous avez un horaire variable, seuls les objectifs quotidiens sont affichés car ils sont plus pertinents pour votre situation."
        case "es": return "Como tienes un horario variable, solo se muestran los objetivos diarios ya que son más relevantes para tu situación."
        default: return "Since you have a variable schedule, only daily targets are shown as they're more relevant for your situation."
        }
    }

    // MARK: - Step 7: How to Use

    var howToUseTitle: String {
        switch language {
        case "fr": return "Comment utiliser l'app"
        case "es": return "Cómo usar la app"
        default: return "How to Use the App"
        }
    }

    var howToUseStepDescription: String {
        switch language {
        case "fr": return "Apprenez le processus étape par étape pour suivre vos pourboires"
        case "es": return "Aprende el proceso paso a paso para rastrear tus propinas"
        default: return "Learn the step-by-step process to track your tips"
        }
    }

    var step1EmployersTitle: String {
        switch language {
        case "fr": return "Créer vos employeurs"
        case "es": return "Crear tus empleadores"
        default: return "Create Your Employers"
        }
    }

    var step1EmployersDescription: String {
        switch language {
        case "fr": return "Commencez par ajouter vos employeurs dans l'onglet Employeurs. Cela vous aidera à organiser vos quarts et revenus par lieu de travail."
        case "es": return "Comienza agregando tus empleadores en la pestaña Empleadores. Esto te ayudará a organizar tus turnos e ingresos por lugar de trabajo."
        default: return "Start by adding your employers in the Employers tab. This helps organize your shifts and income by workplace."
        }
    }

    var step2CalendarTitle: String {
        switch language {
        case "fr": return "Créer vos quarts dans le calendrier"
        case "es": return "Crear turnos en el calendario"
        default: return "Create Shifts in Calendar"
        }
    }

    var step2CalendarDescription: String {
        switch language {
        case "fr": return "Utilisez l'onglet Calendrier pour créer des quarts de travail. Définissez vos heures, dates et employeur pour chaque quart."
        case "es": return "Usa la pestaña Calendario para crear turnos de trabajo. Define tus horas, fechas y empleador para cada turno."
        default: return "Use the Calendar tab to create work shifts. Set your hours, dates, and employer for each shift."
        }
    }

    var step3EntriesTitle: String {
        switch language {
        case "fr": return "Ajouter des entrées de pourboires"
        case "es": return "Agregar entradas de propinas"
        default: return "Add Tip Entries"
        }
    }

    var step3EntriesDescription: String {
        switch language {
        case "fr": return "Pour chaque quart, tapez dessus pour ajouter vos pourboires, ventes et heures réelles. C'est ici que vous enregistrez vos gains quotidiens."
        case "es": return "Para cada turno, tócalo para agregar tus propinas, ventas y horas reales. Aquí es donde registras tus ganancias diarias."
        default: return "For each shift, tap it to add your tips, sales, and actual hours. This is where you record your daily earnings."
        }
    }

    var step4DashboardTitle: String {
        switch language {
        case "fr": return "Voir vos statistiques dans le tableau de bord"
        case "es": return "Ver estadísticas en el panel"
        default: return "View Stats in Dashboard"
        }
    }

    var step4DashboardDescription: String {
        switch language {
        case "fr": return "Le tableau de bord vous montre un résumé de vos gains par jour, semaine, mois et année. Suivez vos progrès et vos tendances facilement."
        case "es": return "El panel te muestra un resumen de tus ganancias por día, semana, mes y año. Rastrea tu progreso y tendencias fácilmente."
        default: return "The dashboard shows you a rollup of your earnings by day, week, month, and year. Track your progress and trends easily."
        }
    }

    var syncFeaturesTitle: String {
        switch language {
        case "fr": return "Synchronisation multi-appareils"
        case "es": return "Sincronización multi-dispositivo"
        default: return "Multi-Device Sync"
        }
    }

    var syncFeaturesDescription: String {
        switch language {
        case "fr": return "Connectez-vous depuis iPhone, iPad, ou bientôt Android. Toutes vos données se synchronisent automatiquement entre vos appareils."
        case "es": return "Inicia sesión desde iPhone, iPad, o pronto Android. Todos tus datos se sincronizan automáticamente entre dispositivos."
        default: return "Log in from iPhone, iPad, or soon Android. All your data syncs automatically across your devices."
        }
    }

    // MARK: - Step Descriptions

    func currentStepDescription(for step: Int) -> String {
        switch step {
        case 1: return languageStepDescription
        case 2: return multipleEmployersStepDescription
        case 3: return weekStartStepDescription
        case 4: return securityStepDescription
        case 5: return variableScheduleStepDescription
        case 6: return salesAndTipTargetsStepDescription
        case 7: return howToUseStepDescription
        default: return ""
        }
    }

    var defaultEmployerLabel: String {
        switch language {
        case "fr": return "Employeur par défaut"
        case "es": return "Empleador predeterminado"
        default: return "Default Employer"
        }
    }

    var selectEmployerPlaceholder: String {
        switch language {
        case "fr": return "Sélectionner un employeur"
        case "es": return "Seleccionar empleador"
        default: return "Select Employer"
        }
    }
}

File: ProTip365Android/app/src/main/java/com/protip365/app/presentation/localization/OnboardingLocalization.kt
package com.protip365.app.presentation.localization

class OnboardingLocalization(private val language: String) {

    // MARK: - Welcome and General
    val welcomeTitle: String
        get() = when (language) {
            "fr" -> "Configuration initiale"
            "es" -> "Configuración inicial"
            else -> "Initial Setup"
        }

    val backButtonText: String
        get() = when (language) {
            "fr" -> "Retour"
            "es" -> "Atrás"
            else -> "Back"
        }

    val nextButtonText: String
        get() = when (language) {
            "fr" -> "Suivant"
            "es" -> "Siguiente"
            else -> "Next"
        }

    val finishButtonText: String
        get() = when (language) {
            "fr" -> "Terminer"
            "es" -> "Finalizar"
            else -> "Finish"
        }

    val skipButtonText: String
        get() = when (language) {
            "fr" -> "Passer"
            "es" -> "Omitir"
            else -> "Skip"
        }

    val getStartedText: String
        get() = when (language) {
            "fr" -> "Commencer"
            "es" -> "Comenzar"
            else -> "Get Started"
        }

    // MARK: - Step 1: Language Selection
    val languageStepTitle: String
        get() = when (language) {
            "fr" -> "Choisissez votre langue"
            "es" -> "Elige tu idioma"
            else -> "Choose Your Language"
        }

    val languageStepDescription: String
        get() = when (language) {
            "fr" -> "Sélectionnez votre langue préférée pour l'interface de l'application"
            "es" -> "Selecciona tu idioma preferido para la interfaz de la aplicación"
            else -> "Select your preferred language for the app interface"
        }

    val englishText: String
        get() = when (language) {
            "fr" -> "Anglais"
            "es" -> "Inglés"
            else -> "English"
        }

    val frenchText: String
        get() = when (language) {
            "fr" -> "Français"
            "es" -> "Francés"
            else -> "French"
        }

    val spanishText: String
        get() = when (language) {
            "fr" -> "Espagnol"
            "es" -> "Español"
            else -> "Spanish"
        }

    // MARK: - Step 2: Multiple Employers
    val multipleEmployersStepTitle: String
        get() = when (language) {
            "fr" -> "Employeurs multiples?"
            "es" -> "¿Múltiples empleadores?"
            else -> "Multiple Employers?"
        }

    val multipleEmployersStepDescription: String
        get() = when (language) {
            "fr" -> "Travaillez-vous pour plusieurs employeurs ou entreprises?"
            "es" -> "¿Trabajas para múltiples empleadores o empresas?"
            else -> "Do you work for multiple employers or businesses?"
        }

    val yesMultipleEmployersText: String
        get() = when (language) {
            "fr" -> "Oui, je travaille pour plusieurs employeurs"
            "es" -> "Sí, trabajo para múltiples empleadores"
            else -> "Yes, I work for multiple employers"
        }

    val trackShiftsSeparatelyText: String
        get() = when (language) {
            "fr" -> "Suivre les quarts et pourboires séparément pour chaque employeur"
            "es" -> "Rastrear turnos y propinas por separado para cada empleador"
            else -> "Track shifts and tips separately for each employer"
        }

    val noSingleEmployerText: String
        get() = when (language) {
            "fr" -> "Non, je travaille pour un seul employeur"
            "es" -> "No, trabajo para un solo empleador"
            else -> "No, I work for one employer"
        }

    val simplifiedTrackingText: String
        get() = when (language) {
            "fr" -> "Suivi simplifié pour un seul employeur"
            "es" -> "Seguimiento simplificado para un solo empleador"
            else -> "Simplified tracking for single employer"
        }

    // MARK: - Step 3: Week Start
    val weekStartStepTitle: String
        get() = when (language) {
            "fr" -> "Jour de début de semaine"
            "es" -> "Día de inicio de semana"
            else -> "Week Start Day"
        }

    val weekStartStepDescription: String
        get() = when (language) {
            "fr" -> "Quand commence votre semaine de travail?"
            "es" -> "¿Cuándo comienza tu semana laboral?"
            else -> "When does your work week begin?"
        }

    val sundayText: String
        get() = when (language) {
            "fr" -> "Dimanche"
            "es" -> "Domingo"
            else -> "Sunday"
        }

    val mondayText: String
        get() = when (language) {
            "fr" -> "Lundi"
            "es" -> "Lunes"
            else -> "Monday"
        }

    // MARK: - Step 4: Security
    val securityStepTitle: String
        get() = when (language) {
            "fr" -> "Paramètres de sécurité"
            "es" -> "Configuración de seguridad"
            else -> "Security Settings"
        }

    val securityStepDescription: String
        get() = when (language) {
            "fr" -> "Protégez vos données financières avec les fonctionnalités de sécurité"
            "es" -> "Protege tus datos financieros con funciones de seguridad"
            else -> "Protect your financial data with security features"
        }

    val noSecurityText: String
        get() = when (language) {
            "fr" -> "Aucune sécurité"
            "es" -> "Sin seguridad"
            else -> "No Security"
        }

    val noAdditionalSecurityText: String
        get() = when (language) {
            "fr" -> "Aucune sécurité supplémentaire"
            "es" -> "Sin seguridad adicional"
            else -> "No additional security"
        }

    val pinProtectionText: String
        get() = when (language) {
            "fr" -> "Protection par code PIN"
            "es" -> "Protección con PIN"
            else -> "PIN Protection"
        }

    val protectWithPinText: String
        get() = when (language) {
            "fr" -> "Protéger avec un code PIN à 4 chiffres"
            "es" -> "Proteger con un PIN de 4 dígitos"
            else -> "Protect with a 4-digit PIN"
        }

    val biometricProtectionText: String
        get() = when (language) {
            "fr" -> "Protection biométrique"
            "es" -> "Protección biométrica"
            else -> "Biometric Protection"
        }

    val useBiometricText: String
        get() = when (language) {
            "fr" -> "Utiliser la reconnaissance d'empreinte digitale ou faciale"
            "es" -> "Usar reconocimiento de huella dactilar o facial"
            else -> "Use fingerprint or face recognition"
        }

    // MARK: - Step 5: Variable Schedule
    val scheduleStepTitle: String
        get() = when (language) {
            "fr" -> "Type d'horaire"
            "es" -> "Tipo de horario"
            else -> "Schedule Type"
        }

    val scheduleStepDescription: String
        get() = when (language) {
            "fr" -> "Avez-vous un horaire de travail variable ou fixe?"
            "es" -> "¿Tienes un horario de trabajo variable o fijo?"
            else -> "Do you have a variable or fixed work schedule?"
        }

    val variableScheduleText: String
        get() = when (language) {
            "fr" -> "Horaire variable"
            "es" -> "Horario variable"
            else -> "Variable Schedule"
        }

    val hoursChangeWeeklyText: String
        get() = when (language) {
            "fr" -> "Mes heures de travail changent d'une semaine à l'autre"
            "es" -> "Mis horarios de trabajo cambian de semana a semana"
            else -> "My work hours change from week to week"
        }

    val fixedScheduleText: String
        get() = when (language) {
            "fr" -> "Horaire fixe"
            "es" -> "Horario fijo"
            else -> "Fixed Schedule"
        }

    val consistentHoursText: String
        get() = when (language) {
            "fr" -> "Mes heures de travail sont constantes chaque semaine"
            "es" -> "Mis horarios de trabajo son consistentes cada semana"
            else -> "My work hours are consistent each week"
        }

    // MARK: - Step 6: Targets
    val targetsStepTitle: String
        get() = when (language) {
            "fr" -> "Définissez vos objectifs"
            "es" -> "Establece tus objetivos"
            else -> "Set Your Goals"
        }

    val targetsStepDescription: String
        get() = when (language) {
            "fr" -> "Définissez des objectifs quotidiens pour suivre vos progrès"
            "es" -> "Establece objetivos diarios para rastrear tu progreso"
            else -> "Set daily targets to track your progress"
        }

    val tipTargetPercentText: String
        get() = when (language) {
            "fr" -> "Objectif de pourboire %"
            "es" -> "Objetivo de propina %"
            else -> "Tip Target %"
        }

    val dailySalesTargetText: String
        get() = when (language) {
            "fr" -> "Objectif de ventes quotidien"
            "es" -> "Objetivo de ventas diario"
            else -> "Daily Sales Target"
        }

    val dailyHoursTargetText: String
        get() = when (language) {
            "fr" -> "Objectif d'heures quotidien"
            "es" -> "Objetivo de horas diario"
            else -> "Daily Hours Target"
        }

    // MARK: - Step 7: Completion
    val completionStepTitle: String
        get() = when (language) {
            "fr" -> "🎉 Vous êtes prêt!"
            "es" -> "🎉 ¡Estás listo!"
            else -> "🎉 You're All Set!"
        }

    val completionStepDescription: String
        get() = when (language) {
            "fr" -> "Bienvenue sur ProTip365! Vous êtes prêt à commencer à suivre vos pourboires et gains."
            "es" -> "¡Bienvenido a ProTip365! Estás listo para comenzar a rastrear tus propinas y ganancias."
            else -> "Welcome to ProTip365! You're ready to start tracking your tips and earnings."
        }

    val whatsNextText: String
        get() = when (language) {
            "fr" -> "Et maintenant?"
            "es" -> "¿Qué sigue?"
            else -> "What's Next?"
        }

    val startAddingShiftsText: String
        get() = when (language) {
            "fr" -> "• Commencer à ajouter vos quarts et pourboires"
            "es" -> "• Comenzar a agregar tus turnos y propinas"
            else -> "• Start adding your shifts and tips"
        }

    val viewEarningsDashboardText: String
        get() = when (language) {
            "fr" -> "• Voir vos gains sur le tableau de bord"
            "es" -> "• Ver tus ganancias en el panel"
            else -> "• View your earnings on the dashboard"
        }

    val trackProgressGoalsText: String
        get() = when (language) {
            "fr" -> "• Suivre vos progrès vers les objectifs"
            "es" -> "• Rastrear tu progreso hacia los objetivos"
            else -> "• Track your progress toward goals"
        }

    val exportDataAnytimeText: String
        get() = when (language) {
            "fr" -> "• Exporter vos données à tout moment"
            "es" -> "• Exportar tus datos en cualquier momento"
            else -> "• Export your data anytime"
        }
}






